config/tpmcalculator.yaml

```diff
@@ -0,0 +1,7 @@
+project_top_level: /SAN/vyplab/TDP43_RNA/IPSC_WARD_RIBOSEQ/
+bam_subfolder: bams/
+suffix: ".readlen_filtered"
+end_type: se
+tpmcalculator_output_folder: TPMcalculator/
+reference_gtf: /SAN/vyplab/vyplab_reference_genomes/annotation/human/GRCh38/gencode.v40.annotation.gtf
+tpmcalculator_path: /SAN/vyplab/alb_projects/tools/TPMCalculator/bin/TPMCalculator
```

Comment:
Would rather not have the tpmcalculator_path, it goes against how we want our new single steps to be designed. Personally think it would be unnecessary to add to conda env as it's so specific, but could alternatively pull the [biocontainer](https://quay.io/repository/biocontainers/tpmcalculator?tab=tags) and let snakemake do it's thing. Bonus is it's much quicker than building a new conda env

also should be named like `config/tpmcalculator_config.yaml` so naming is consistent across our new single steps & the general submit script